proto/buf.yaml

```diff
@@ -0,0 +1,22 @@
+version: v1
+name: buf.build/celestiaorg/celestia-app
+deps:
+  - buf.build/cosmos/cosmos-proto
+  - buf.build/cosmos/gogo-proto
+  - buf.build/googleapis/googleapis
+breaking:
```

Comment:
[note to reviewers]
This is using the same breaking and lint specified in the cosmos-sdk 0.46